app/assets/stylesheets/pages/_docs.scss

```diff
@@ -1,13 +1,33 @@
-$color-green: rgb(3, 165, 98);
+@mixin margin-left-sidebar {
+  margin-left: $sidebar-width + ($gutter / 2);
+}
 
-.Page {
```

Comment:
It looks like you've brought back a bunch of styles that I had removed. We might need to revisit some of the merge conflicts.

Reply:
Ah yes, this was on purpose. I reworked this to achieve a few things (forgot to leave comments in the commit):

1. Moved spacing to header from `.Page` to `.PageContainer`
2. I don't think we should @extend `.TextContent` to `.Page`.

Might be easier to sync up to chat about this one.

Reply:
We paired and resolved this.